apex-squid/src/main/java/org/fundacionjala/enforce/sonarqube/apex/api/grammar/buildersource/Declaration.java

```diff
@@ -2,7 +2,6 @@
  * Copyright (c) Fundacion Jala. All rights reserved.
```

Comment:
@JimmyRojas Mmm yeah I was considering adding the recovery functionality for the properties too because their braces ("{}") were causing some issues (the Block statement recovery wasn't handling them), but I removed it because in case there would be an error for a property it would be skipped and handled by the Declaration recovery, so it wasn't necessary